integration-tests/actions/actions.go

```diff
@@ -24,6 +24,17 @@ import (
 	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
 )
 
+var (
```

Comment:
Actions seems like a bad place to put this. Maybe another package for common network configs?

Reply:
moved to `config` package